test/wrapper.gradle

```
@@ -1,3 +1,3 @@
 wrapper {
-    gradleVersion = '4.4.1'
+    gradleVersion = '4.6'
```

```suggestion
    gradleVersion = '4.6.0'
```

Does this need to be `4.6.0`? In the previous file `bin/templates/project/app/build.gradle` you had written it as 

```
wrapper {
    gradleVersion = '4.6.0'
}
```
but this file was different.

---

Ahh, right... so this is used to determine which version of gradle to download, and there is no `4.6.0` release, there is only a `4.6` release.